yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -718,12 +782,19 @@ class FlatMapOperator : public FlowableOperator<T, R, FlatMapOperator<T, R>> {
           : flatMapSubscription_(std::move(subscription)) {}
 
       void onSubscribeImpl() final {
+#ifdef NDEBUG
```

Comment:
Did you mean `#ifndef NDEBUG`?  For this, and all follow up uses?  You want the debugging code in the cases where NDEBUG is _not defined_ right, or am I getting myself all flipped up.